plinth/modules/jsxc/urls.py

```diff
@@ -0,0 +1,29 @@
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+URLs for the JSXC module
+"""
+
+from django.conf.urls import url
+
+from .views import JSXCServiceView, JsxcView
+
+
+urlpatterns = [
+    url(r'^apps/jsxc/$', JsxcView.as_view(), name='jsxc'),
```

Comment:
Need 2 URLs:
- apps/jsxc for JSXCServiceView ('index')
- apps/jsxc/jsxc for JsxcView ('jsxc')